projects/ngrx.io/content/marketing/docs.md

```diff
@@ -2,6 +2,44 @@
 
 NgRx is a framework for building reactive applications in Angular. NgRx provides state management, isolation of side effects, entity collection management, router bindings, code generation, and developer tools that enhance developers experience when building many different types of applications.
 
+## Why NgRx?
+
+NgRx is built for creating maintainable explicit applications, by encouraging the use of actions in order to express state changes.
+
+If you're building an application that has a lot of user interaction, many different event sources that's where NgRx is going to really shine in your application.
+
+
+### Serializability
+
+By normalizing state changes and pass them through observables NgRx provides serializability and good control of the application. This enables to save and serialize states to the `localStorage` or any other external storage.
+In addition, it also allows to inspect, download, upload, and dispatch actions, as well as time, travel all from the [Store Devtools](guide/store-devtools).
```

Comment:
time travel is an oversold item that brings questionable value. Let's not make is a point in "Why NgRx" 🙊
```suggestion
In addition, it also allows to inspect, download, upload, and dispatch actions, all from the [Store Devtools](guide/store-devtools).
```

Reply:
I like it very much but I see your point 😆 😆